chatbees_script.js

```diff
@@ -79,18 +80,15 @@ function chatbeesSendMessage() {
 
   // get the answer from the service
   const apiUrl = 'https://' + aid + '.us-west-2.aws.chatbees.ai/docs/ask';
-  jsonData = JSON.stringify({namespace_name: 'public', collection_name: collectionName, question: userInput});
+  jsonData = JSON.stringify({namespace_name: namespaceName, collection_name: collectionName, question: userInput});
   if (historyMessages.length > 0) {
-    jsonData = JSON.stringify({namespace_name: 'public', collection_name: collectionName, question: userInput, history_messages: historyMessages});
+    jsonData = JSON.stringify({namespace_name: namespaceName, collection_name: collectionName, question: userInput, history_messages: historyMessages});
   }
 
   fetch(apiUrl, {
     method: 'POST',
     headers: {
-      'Content-Type': 'application/json',
-      // Please replace the api-key with a valid API Key you created.
-	  // This is not required soon. Publishing a collection will be supported soon.
-	  'api-key': 'Replace with your API Key',
+      'Content-Type': 'application/json'
```

Comment:
let's not remove it. comment out the api-key, and update comments to like: if the collection does not allow public read, please add your api-key here.